web/packages/teleport/src/SessionRecordings/view/ProgressBar/ProgressBar.tsx

```diff
@@ -53,6 +84,18 @@ export default function ProgressBar(props: ProgressBarProps) {
         />
       </SliderContainer>
       <Restart onRestart={props.onRestart} />
+      {showTimeline && (
+        <HoverTooltip tipContent="Show Timeline">
+          <ActionButton
+            style={{
+              marginLeft: '16px',
```

Comment:
Let's use `space[3]`

Reply:
This is going soon too and I don't really want to import the theme to use it here - this is copying the `Restart` button